.changeset/long-walls-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+Finalized Hds::Table component
```

Comment:
Can you add a more detailed entry here so consumers know what we did and also link back to the original PR(s) where the changeset would have been

Reply:
I think that would go in the version PR, yes? Don't these changelogs get deleted? 

(I'm asking because I wasn't sure how to do it, so I went back and looked at previous files)

Reply:
They do, but the content in the changelog is taken from here as part of that process. 

It _is_ possible to manually adjust the changelog in the version PR, but that should be an exception.